### ISSUE TITLE
Issue: Increased Harborarea Manola

### ISSUE THREAD

**Issue by JanDerNixKann** [anno-mods/Spice-it-Up#40]
Hiho, I have a quick question regarding the "Increased Harborarea" mod. I noticed that on Manola Island there seems to be an overlap between the two adjacent harbour areas. This creates a small area that cannot be built on. Unfortunately, this is in a rather stupid location (see attachment). Is it possible to fix this? If not, no problem. I just thought I'd ask.

Either way, thank you for all your hard work.

![2023-09-01 17_36_56-Anno 1800](https://github.com/anno-mods/Spice-it-Up/assets/143826616/86e1e8a2-1dce-482e-9261-f93624d519bf)

**Comment by taubenangriff** [comment]
Nah, can't change. You can try playing around with the harbor radius, but you probably will still get this kind of blocked area still.
